docs/resources/gateway.md

```diff
@@ -19,6 +19,7 @@ resource "valtix_gateway" "aws-gw1" {
   instance_type           = "AWS_M5_2XLARGE"
   gateway_image           = "2.11-08"
   gateway_state           = "ACTIVE"
+  wait_for_gateway_state  = "ACTIVE"

```

Comment:
wait_for_gateway_state is a true/false that waits until the given gateway_state is achieved. This defaults to true and makes it a sync action. If the user wants async action (like UI) they can set the value to false